src/config/test/config_tests.erl

```diff
@@ -441,8 +437,8 @@ should_write_changes(_, _) ->
         ?assertEqual("5986", config:get("httpd", "port")),
         ?assertEqual(ok, config:set("httpd", "port", "8080")),
         ?assertEqual("8080", config:get("httpd", "port")),
-        ?assertEqual(ok, config:delete("httpd", "bind_address", "8080")),
-        ?assertEqual(undefined, config:get("httpd", "bind_address"))
+        ?assertEqual(ok, config:delete("httpd", "bind_address")),
+        ?assertEqual("127.0.0.1", config:get("httpd", "bind_address"))
```

Comment:
Why does not this remain `undefined`?

Reply:
It was `undefined` before because the `config:delete("httpd", "bind_address")` on the previous line removed it from the `ets` table, but did not reload the default. With this change, after deletion the default gets reloaded from disk and inserted into the ets table. In short, It is the crux of this PR.

Reply:
Okay, that is what I thought.  Thanks for the confirmation.